### DISCUSSION TITLE
How can disable variabel --b1?

### DISCUSSION THREAD

**Discussion by jjhonny** [saadeghi/daisyui#3038] [Q&A] [answered]
I think I need to disable the --b1 variable that comes standard in the library because when a component passes 100% of the screen it applies this variable as the background of my screen, the problem is that my background is a different color from this variable -- b1 which is applied automatically.
![Captura de tela 2024-05-16 103755](https://github.com/saadeghi/daisyui/assets/124160277/d07ba759-1bcd-4f65-ab4d-82dfd791732f)

**Comment by saadeghi** [comment]
disabling the CSS variable is not the solution.  
Ideally your root element height should contain all your content and you can do it by adding a height value to it.  
Also, you can add a background color to the root element if you want it to be a different color. You can do this in CSS or using a `bg-*` class.

I could help better if you provide a code example because I'm not sure how your HTML structure looks like.

Let me know if you have any questions.

**Reply by jjhonny** [reply]
![Captura de tela 2024-05-16 142531](https://github.com/saadeghi/daisyui/assets/124160277/9de9f168-ee29-4777-bd5d-833648902958)
![Captura de tela 2024-05-16 142614](https://github.com/saadeghi/daisyui/assets/124160277/d99586a1-082a-4be5-b49c-807b469edf0c)
Are these images useful?

**Comment by saadeghi** [comment, marked as answer]
Instead of giving background color and min-height to the `body` give it to `html`

body is not the document root.

**Comment by jjhonny** [comment]
Thank you very much for your help, the problem really was that I was passing h-screen to my element and the right thing to do was to pass min-h-screen. Thank you for your attention.
